lib/network_test.go

```diff
@@ -189,38 +196,38 @@ func TestHeaderConversionAndReadWriteMessage(t *testing.T) {
 	// versions we want to test.
 	for _, expectedBlockHeader := range expectedBlockHeadersToTest {
 		data, err := expectedBlockHeader.ToBytes(false)
-		assert.NoError(err)
+		require.NoError(err)
```

Comment:
Why not pass in the testing instance `t` to the NoError function? Same comment for all of uses of require below
```suggestion
		require.NoError(t, err)
```